webpack/localization-plugin/src/interfaces.ts

```diff
@@ -153,9 +153,10 @@ export interface ILocalizationPluginOptions {
   localizedData: ILocalizedData;
 
   /**
-   * This option is used to specify `.resx` and `.loc.json` files that should not be processed by this plugin.
+   * This option is used to specify `.resx`, `.resx.json` and `.loc.json` files that should not be processed by
+   * this plugin.
    */
-  filesToIgnore?: string[];
+  globsToIgnore?: string[];
```

Comment:
It would be helpful if the documentation could give an idea of why someone would use this.  (Why would your filename be `.loc.json` but your do NOT want it to be localized?)

Reply:
(It would also be helpful if the docs showed a realistic looking example glob.)